Saule/JsonApiSerializerOfT.cs

```diff
@@ -46,6 +47,11 @@ public JsonApiSerializer()
         public bool Paginate { get; set; } = false;
 
         /// <summary>
+        /// True if responses should be paginated, otherwise false.
```

Comment:
Perhaps, but it (the property) also does not seem to be used and I can't see what it's purpose was intended for. Will remove it.